src/excmds.ts

```diff
@@ -2077,57 +2120,63 @@ export async function yank(...content: string[]) {
 
     If `excmd == "yankmd"`, copy the title and url of the open page formatted in Markdown for easy use on sites such as reddit.
 
-    Unfortunately, javascript can only give us the `clipboard` clipboard, not e.g. the X selection clipboard.
+    If you're on Linux and the native messenger is installed, Tridactyl will call an external binary (either xclip or xsel) to read or write to your X selection buffer.
+
+    When doing a read operation (i.e. open or tabopen), if "externalclipboard" is set to "only" or "first", the X selection buffer will be read instead of the clipboard. Set "externalclipboard" to "last" or "never" to use the clipboard.
+
+    When doing a write operation, if "externalclipboard" is set to "only", only the X selection buffer will be written to. If "externalclipboard" is set to "first" or "last", both the X selection and the clipboard will be written to. If "externalclipboard" is set to "never", only the clipboard will be written to.
```

Comment:
Could you document externalclipboardcmd and the flags it must support? It's plausible that some people might want to use a complicated clipboard manager.